tools/ci_build/github/azure-pipelines/templates/common-variables.yml

```diff
@@ -1,3 +1,7 @@
 variables:
-  common_cuda_version: '11.8'
```

Comment:
This variable was used by big model CI only (Its stable diffusion demo depends on cuda 11.8)
Since now cuda 12 becomes the default version for all non-pkg CIs, I remove this variable to avoid confusion and hardcode it to big model CI yml